ckanext/ontario_theme/plugin.py

```diff
@@ -394,6 +394,18 @@ def get_keyword_count(keyword_lang, language):
 
     return keyword_count_by_lang
 
+def sort_page_items(obj, fields_dict):
+    '''Helper to return the page items object sorted by title_translated
+    for the current language.
+    '''
+    field = fields_dict['field']
+    lang = fields_dict['lang']
+    reverse = fields_dict['reverse']
+    return sorted(obj, 
+                  key=lambda x: x[field][lang], 
```

Comment:
I'm getting a `KeyError: 'en'` from this line, when trying to go to the next page and when I try choosing the order by "Name Ascending" option on the Ministries search page.

Reply:
I added a check for empty `title_translated` fields and no longer see this error. Please check.